### ISSUE TITLE
~/.local/lib/python3.10/site-packages/rdkit-stubs/Chem/rdMolDescriptors.pyi:185: error: Duplicate argument "self" in function definition

### ISSUE THREAD

**Issue by UnixJunkie** [rdkit/rdkit#7554]
**Describe the bug**
while running mypy on a Python rdkit script, I got the reported error

**To Reproduce**
mypy some_rdkit_script.py

**Expected behavior**
no such error

**Screenshots**
N/A

**Configuration (please complete the following information):**
 - RDKit version:  '2023.09.6'
 - OS: irrelevant
 - Python version (if relevant): 3.10.12
 - Are you using conda? Thanks god, no.
 - If you are using conda, which channel did you install the rdkit from? N/A
 - If you are not using conda: how did you install the RDKit? pip3 install rdkit

**Additional context**
I might send a patch in 5 min.

**Comment by UnixJunkie** [comment]
The PR is ready for review and merging.
Thanks!

**Comment by ElricleNecro** [comment]
This problem is still present in RDKit 2024.3.5, making it impossible to using in project using mypy.
In the file `rdkit-stubs/Chem/rdMolDescriptors.pyi`, in this declaration :
```python
    def __init__(self, self: typing.Any, name: str, version: str) -> None:
        """
            C++ signature :
                void __init__(_object*,_object*,std::string,std::string)
        """
```
there is 2 `self` where only one should be needed:
```python
    def __init__(self: typing.Any, name: str, version: str) -> None:
        """
            C++ signature :
                void __init__(_object*,_object*,std::string,std::string)
        """
```

**Comment by ptosco** [comment]
I am closing this as it has been fixed by #7644, which has already been merged.
